index.html

```diff
@@ -43,8 +66,18 @@ <h1>Awesome Books</h1>
         </thead>
         <tbody id="book-list"></tbody>
       </table>
-    </div>
 
+      <div id="contact" class="d-none mt-5">
+        <div class="d-flex flex-column align-items-center">
+          <h2>contact us</h2>
+          <p>contact@gmail.com</p>
+        </div>
+      </div>
+      <div class="copyright"></div>
+      <div class="card container mt-5 card-footer">
+        <p>&#169; Copyright by Hussein</p>
+      </div>
+    </div>
```

Comment:
### OPTIONAL


- [ ] It would be better for your footer to be fixed to the bottom of your page, Currently the footer is not fixed to the bottom 👍🏿